bundles/basic_modes/mode_definitions.moon

```diff
@@ -299,9 +299,8 @@ common_auto_pairs = {
     comment_syntax: '#'
     indentation: {
       more_after: {
-        { ':%s*$', 'else:%s*$' }
+        { ':%s*$' }
       }
-      less_for: { 'else:%s*$'}
```

Comment:
`less_for` controls what statements should be indented less  compared to the previous line. This change removes this for `else:`.